box/boxunreal.py

```diff
@@ -110,7 +112,8 @@ def save_image(self, action: Action) -> None:
 
         # Generate the next filename
         image_filepath = (
-            f"{self.dataset_path}/" f"{self.images_saved:06}_{str(action).lower()}.png"
+            f"{self.dataset_path}/"
+            f"{self.images_saved:06}_{str(action).lower()}.{str(self.image_ext).lower()}"
```

Comment:
I'd like the filename to use the angle to the target instead of the action name.